TodoWithReact/README.md

```diff
@@ -0,0 +1,51 @@
+Title : React front end app for todoapp.
+
+Introduction : The project's aim is to provide the UI for different api calls of todoapp.
+
+Launch : 5th nov 2019
+
+Project status : completed
+
+Example of use : To retrieve all todos fron the database.
+
+Size of project : 8.9kb
+
+Platform Used : Ubuntu 16.04
+
+Technologies :  
+    a) React
+    b) React-Router
+    c) React-axios
+
+Applications needed : 
+    1) npm
+    2) node 
+    3) Backend which provides api's for all todo app on localhost:8080.
+
+Softwares used : 
+    a) Visual studio code.
+    b) npm (6.11.3)
+    c) node (V12.10.0)
+
+To run this project : 
+1) Extract the zip in visual studio's workspace.
+2) Open the project in visual studio code.
+3) Open terminal.
+4) Execute following steps : 
```

Comment:
do we need to execute these steps?will plain `npm install` wont work because they are defined in the package.json?